ST_params.c

```diff
@@ -1107,7 +1090,7 @@ static void _species_init( void) {
       Species[sp]->received_prob = 0;
       Species[sp]->cohort_surv = cohort;
       Species[sp]->var = var;
-      Species[sp]->pseed = pseed / Globals.plotsize;
+      Species[sp]->pseed = pseed / Globals->plotsize;
 /*      Species[sp]->ann_mort_prob = (age > 0)
                                          ? -log(cohort)/age
                                          : 0.0;
```

Comment:
Can we get rid of this commented-out code?

Reply:
@kpalmqui Is this line something we can remove?

Reply:
@chaukap @fdpierson I would liked to retain this for now despite the fact that it is not being used. This appears to be related to establishment of annuals, which we need to revisit. I will open a new issue for this.

Reply:
https://github.com/DrylandEcology/STEPWAT2/issues/381